packages/app-project/src/components/ProjectAnnouncement/ProjectAnnouncement.js

```diff
@@ -18,18 +26,44 @@ const components = {
 }
 
 function ProjectAnnouncement (props) {
-  const { announcement } = props
+  const { announcement, closeFn } = props
+
   return (
-    <Box align='center' background='neutral-4' fill='horizontal' pad='small'>
-      <Markdownz components={components}>
-        {announcement}
-      </Markdownz>
-    </Box>
+    <>
+      <Media at='default'>
+        <Box align='center' background='neutral-4' fill='horizontal' pad='small'>
+          <Box align='center' direction='row' gap='small' justify='between'>
+            <Markdownz components={components}>
+              {announcement}
+            </Markdownz>
+            <CloseButton closeFn={closeFn} />
+          </Box>
+        </Box>
+      </Media>
+
+      <Media greaterThan='default'>
+        <Box align='center' background='neutral-4' fill='horizontal' pad='small'>
+          <Box align='center' direction='row' gap='small' justify='between' width='xxlarge'>
+            <Balancer />
+            <Box align='center' fill='horizontal'>
+              <Markdownz components={components}>
+                {announcement}
+              </Markdownz>
+            </Box>
+            <CloseButton closeFn={closeFn} />
+          </Box>
+        </Box>
+      </Media>
+    </>
```

Comment:
This seems a bit complicated. It seems like this could be simplified using flex box, but I'd have to play around with it to figure out what to suggest.

Reply:
I know. The aim was to get the text centred to the viewport, with an off-centre close button, that doesn't use absolute positioning (no weird edge cases where they collide), and that keeps its proportions when text is resized. One for another day, probably

Reply:
I know. The aim was to get the text centred to the viewport, with an off-centre close button, that doesn't use absolute positioning (no weird edge cases where they collide), and that keeps its proportions when text is resized. One for another day, probably

Reply:
Centering stuff is harder than it should be 👍